### ISSUE TITLE
Awesome 4.0

### ISSUE THREAD

**Issue by rogorido** [ivanbalashov239/awesome-revelation#1]
Is this supposed to work with awesome 4?

**Comment by ivanbalashov239** [comment]
@rogorido no, and it is not maintened, as you can see in the commits. May be i will update this one if and when i will use awesome4

**Comment by ivanbalashov239** [comment]
@rogorido and, of course, you a free to fork it and update for yourself.

**Comment by rogorido** [comment]
thanks for your answer!
